### DISCUSSION TITLE
Support for new subscription client graphql-ws

### DISCUSSION THREAD

**Discussion by giautm** [99designs/gqlgen#2033] [Ideas]
The project [`subscriptions-transport-ws`](https://github.com/apollographql/subscriptions-transport-ws) was archived. And they introduce a new project to replace [`graphql-ws`](https://github.com/enisdenjo/graphql-ws) which implements the different protocol.

~So, we need to re-implement the subscription feature to support the new client.~

The new protocol `graphql-transport-ws` is supported since Nov 2021 via this PR: https://github.com/99designs/gqlgen/pull/1507

https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md

**Comment by sgentile** [comment]
I'd also like to bump this - other libraries such as graphql-mesh rely on us using graphql-ws

**Comment by giautm** [comment]
Wow, I really can't believe `graphql-ws` is supported since Nov 2021 by this PR https://github.com/99designs/gqlgen/pull/1507
